mobile/lib/widgets/asset_viewer/bottom_gallery_bar.dart

```diff
@@ -391,10 +391,22 @@ class BottomGalleryBar extends ConsumerWidget {
               backgroundColor: Colors.black.withOpacity(0.4),
               unselectedIconTheme: const IconThemeData(color: Colors.white),
               selectedIconTheme: const IconThemeData(color: Colors.white),
-              unselectedLabelStyle: const TextStyle(color: Colors.black),
-              selectedLabelStyle: const TextStyle(color: Colors.black),
-              showSelectedLabels: false,
-              showUnselectedLabels: false,
+              unselectedLabelStyle: const TextStyle(
```

Comment:
Is this better set in the `immich_app_theme.dart` or is this the only place we should apply this style? There is a `bottomNavigationBarTheme` in the style:

![image](https://github.com/user-attachments/assets/15010c64-144a-4a9a-be75-ad01e30e9b15)

Reply:
This is the only place that is using this style AFAIK